README.md

```diff
@@ -316,6 +316,8 @@ When initializing an autocomplete, there are a number of options you can configu
 * `keyboardShortcuts` - Array of shortcut that will focus the input. For example if you want to bind `s` and `/`
  you can specify: `keyboardShortcuts: ['s', '/']`
 
+* `ariaLabelledBy` - An optional id to use for the `aria-labelledby` attribute. Specify `false` to exclude the attribute. Defaults to using the `placeholder` as the label if a `placeholder` is specified.
```

Comment:
`ariaLabelledBy` would translate to `aria-labelled-by`, so this should probably be `ariaLabelledby` (lowecase B) instead.